spec/app_support/reservations/rendering_spec.rb

```diff
@@ -94,16 +94,26 @@
     end
 
     context "without an order" do
+      subject(:reservation) do
+        build_stubbed(
+          :admin_reservation,
+          reserve_start_at: reserve_start_at,
+          reserve_end_at: reserve_end_at,
+          actual_start_at: actual_start_at,
+          actual_end_at: actual_end_at,
+        )
+      end
+
       context "with details requested" do
         it "returns a hash without extra details about the order" do
           expect(reservation.as_calendar_object(with_details: true))
-            .to eq(base_hash.merge("title" => "Admin Reservation"))
+            .to eq(base_hash.merge("title" => "Admin Hold"))
```

Comment:
Do you clean up the hash syntax in the next PR? Or does it need to be cleaned?

Reply:
Yep, https://github.com/tablexi/nucore-open/pull/1242